populator-machinery/controller_test.go

```diff
@@ -307,10 +335,23 @@ func compareNotifyMap(want []string, got map[string]*stringSet) error {
 	return nil
 }
 
+func compareObjects(want *vpObjects, got *vpObjects) error {
```

Comment:
Can you just compare the entire struct?

Reply:
The `got` object get from the KubeClient is not exactly the same as the `want` object. The k8s API modify the object so I compare the exact field